packages/app/app/containers/SoundContainer/sleeptimer.js

```diff
@@ -0,0 +1,32 @@
+let props;
+let sleeptimerSleepDate;
+let sleeptimerTimeToSleep=0;
```

Comment:
Instead of keeping it here in module scope, this should be moved to a container higher in the tree and kept track of with `useState` or in Redux